src/app/child-dev-project/notes/note-attendance-block/note-attendance-block.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from "@angular/core";
+import { OnInitDynamicComponent } from "../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+import { ViewPropertyConfig } from "../../../core/entity-components/entity-list/EntityListConfig";
+import { Note } from "../model/note";
+import { AttendanceLogicalStatus } from "../../attendance/model/attendance-status";
+
+@Component({
+  selector: "app-note-attendance-block",
+  templateUrl: "./note-attendance-block.component.html",
+  styleUrls: ["./note-attendance-block.component.scss"],
+})
+export class NoteAttendanceBlockComponent implements OnInitDynamicComponent {
+  participantsWithStatus: number;
+
+  constructor() {}
+
+  onInitFromDynamicConfig(config: ViewPropertyConfig) {
+    const note = config.entity as Note;
+    const attendanceStatus = config.config.status as AttendanceLogicalStatus;
+    this.participantsWithStatus = note.countWithStatus(attendanceStatus);
```

Comment:
let's add an `@Input() attendanceStatus` property to the component and assign from the config to that. I would like to have the initFromDynamicConfig always as an optional add-on where we can still use the component directly in the "classic" angular way.